capa/ida/plugin/form.py

```diff
@@ -580,15 +583,15 @@ def load_capa_rules(self):
         rule_path: str = settings.user.get(CAPA_SETTINGS_RULE_PATH, "")
         try:
 
-            def on_load_rule(rule_path, i, total):
-                update_wait_box("loading capa rules from %s (%d of %d)" % (rule_path, i, total))
+            def on_load_rule(_, i, total):
+                update_wait_box("loading capa rules from %s (%d of %d)" % (rule_path, i + 1, total))
                 if ida_kernwin.user_cancelled():
                     raise UserCancelledError("user cancelled")
 
-            self.ruleset_cache = capa.main.get_rules([rule_path], on_load_rule=on_load_rule)
+            return capa.main.get_rules([rule_path], on_load_rule=on_load_rule)
```

Comment:
`load_capa_rules` now returns a `RuleSet` or `None` allowing us to easily load and cache multiple rule sets.